src/components/Footer.astro

```diff
@@ -11,7 +11,8 @@ import ThemeSelect from './ThemeSelect.astro'
 <footer class="docs-footer__container">
   <div class="docs-footer__container__support-links">
     <p>
-      <Fragment set:html={textArrowIcon} />Need help? - <a href="#"
+      <Fragment set:html={textArrowIcon} />Need help? - <a
+        href="https://join.slack.com/t/daytonacommunity/shared_invite/zt-273yohksh-Q5YSB5V7tnQzX2RoTARr7Q"
```

Comment:
Please use `https://go.daytona.io/slack` because the join.slack.com link expires often so we just update it in the go.daytona.io redirect link.